devicetypes/zenwithin/zen-thermostat.src/zen-thermostat.groovy

```diff
@@ -511,7 +517,7 @@ def configure()
       	  //Thermostat - Cluster 201
           "zdo bind 0x${device.deviceNetworkId} 1 1 0x201 {${device.zigbeeId}} {}", "delay 500",
           
-          "zcl global send-me-a-report 0x201 0 0x29 5 300 {3200}", 
+          "zcl global send-me-a-report 0x201 0 0x29 0 300 {3200}",
```

Comment:
We do not want to change the `minReportTime` attribute in this zcl cmd. 
FYI:
`minReportTime - Minimum number of seconds between reports`